source/develop/release-management/features/sla/deploy-hosted-engine-hosts-via-engine.md

```diff
@@ -21,6 +30,21 @@ To un-deploy or remove the hosted engine role of a host the engine passes action
 Supply detailed into on how to get the info from the disk, and what is the host-deploy enums we use
 
 # UI side dialog tab
-TBD
+The UI of 'Add' and 'Reinstall' host has a new side panel named 'Hosted-Engine'. It supplies 3 options for the deployment of a host:
+- 'None' meaning don't touch the hosted-engine components
+- 'Deploy' meaning install the ovirt-hosted-engine-ha rpms, place a custom hosted-engine.conf under /etc and start the service
+- 'Undeploy' Remove the hosted-engine.conf and rpms. That action decomissions a host from the HA cluster of hosted engine.
```

Comment:
For some reason, this looks bad (very large and bold) in the preview - please check.